CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features
 
 - Send debug meta for app start transactions (#3543)
+- Automatically profile app launches (#3529)
```

Comment:
I'm fine to align the naming to what we already have for consistency in this case, but I'd like to point out that being a good platform citizen means conforming to the vocabulary of the _platform_. We should not be using Android terminology in an iOS SDK.

Reply:
Two years ago, we decided to go with app start. While it might be a mistake, I want to keep things consistent. We can discuss renaming everything to app launch in the next major, but for now, I would like to stick to app start for consistency.

Reply:
I disagree, we should be using the terminology of the platforms we operate. We should not be carrying over Android terminology into a Cocoa SDK.

We use "launch" extensively throughout the SDK, with the app start measurement stuff looking more like the exception, not the rule. I don't think we should continue down that road; in fact we should rename the app start measurement APIs to be "app launch measurement".

Reply:
We should use the terminology of the target platform. Terminology from Android should not be used in a Cocoa SDK. This is good platform citizenship and respects our customers by not making things more confusing for them.

We use "launch" in other parts of the SDK as well. The app start measurement API actually looks like the exception to the rule, and I would even recommend we rename it to keep things consistent, instead of trying to force consistency in an approach that is, in my opinion, incorrect.

Reply:
We should use terminology that agrees with each target platform. This is good platform citizenship and shows respect to our customers by keeping this simple and consistent. In general I always push for convenience for customers over convenience for ourselves.

We use "launch" in enough other places in the SDK that it seems like the app start measurement stuff seems like the exception, not the rule. I'd recommend we rename that at some point to get get it consistent with the rest of the API, instead of going any further down a path that is, IMO, incorrect.

Reply:
If we acknowledge it's a mistake, I see no reason to rename it. I think it's highly likely that whatever name gets merged here is what will remain indefinitely. I'd prefer not to put something in we think we'll want to rename later. Less breakages for our customers down the line.

I'm all for consistency, which is why we should be aligning with the platform terminology in the first place. We should be respecting the convenience and cognitive burdens of our customers over our own.

Reply:
The feature is still experimental so that we could rename it later. Let's start a separate discussion so that topic doesn't block this PR.